tool/tsh/tsh.go

```diff
@@ -724,7 +729,15 @@ func onListNodes(cf *CLIConf) {
 		return nodes[i].GetHostname() < nodes[j].GetHostname()
 	})
 
-	showNodes(nodes, cf.Verbose)
+	// verbose option will only function if output format is left as default (table)
+	if cf.Verbose && cf.OutputFormat == teleport.Table {
+		cf.OutputFormat = teleport.Verbose
+	}
```

Comment:
Why bother creating a separate `Verbose` format?  Seems to just propagate the value of `cf.Verbose` to `printNodes` directly.